extension/js/add_input_icon.js

```diff
@@ -63,6 +63,14 @@ function addRelayIconToInput(emailInput) {
   };
 
   buttonEl.addEventListener("click", async (e) => {
+    browser.runtime.sendMessage({
+      method: "sendMetricsEvent",
+      eventData: {
+        category: "Addon Input Icon",
+        action: "click",
+      },
+    })
```

Comment:
@lesleyjanenorton so, this would be the call to send event pings to the back-end from the add-on. Look good?